src/autoinstrumentation.ts

```diff
@@ -124,7 +126,8 @@ export const startOpenTelemetryAgent = (distroName: string, opampServerHost: str
     agentDescriptionNonIdentifyingAttributes: {},
     onNewRemoteConfig: (remoteConfig: RemoteConfig) => {
       const resource = localResource
-        .merge(remoteConfig.sdk.remoteResource);
+        .merge(remoteConfig.sdk.remoteResource)
+        .merge(new Resource({ [PROCESS_VPID]: process.pid }));
```

Comment:
I think it makes more sense to add it to the `staticResource` above.
this function will be called any time a new config is received, and this resource is static we can add it just once.